migration_original/ODS1Stage/tables/Show/SOLRTreatmentEntryLevel/spu_translated_SOLRTreatmentEntryLevel.sql

```diff
@@ -0,0 +1,124 @@
+
+-- 1. spuSOLRTreatmentEntryLevel (validated in snowflake)
+CREATE OR REPLACE PROCEDURE SHOW.SP_LOAD_SOLRTREATMENTENTRYLEVEL() 
```

Comment:
Add database name to the procedure name.
```suggestion
CREATE OR REPLACE PROCEDURE ODS1STAGE.SHOW.SP_LOAD_SOLRTREATMENTENTRYLEVEL() 
```

Reply:
Fixed it